src/Avalonia.Controls/Platform/ITopLevelImpl.cs

```diff
@@ -20,12 +20,12 @@ namespace Avalonia.Platform
     public interface ITopLevelImpl : IDisposable
     {
         /// <summary>
-        /// Gets the client size of the toplevel.
```

Comment:
I think toplevel is a proper term for this

Reply:
I was in doubt about that one. Anyway, reverted.